src/data/waypoint.json

```diff
@@ -79,7 +79,7 @@
 		}
 	],
 	"integrationsConfig": {
-		"enabled": false,
+		"enabled": true,
```

Comment:
I've enabled `waypoint` for now, as otherwise I couldn't find an easy way to test these changes, since Vault doesn't seem to have integrations that render into the affected "readme" and "component' views.

🚧 Reminder to self: I'll remove this before merging this PR, it's here for testing purposes only.

Reply:
Good point, and great callout ❇️